inference-engine/src/transformations/include/transformations/convert_opset4_to_opset3/convert_sequences_to_sequences_ie.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+#include <string>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertLSTMSequenceMatcher;
+class TRANSFORMATIONS_API ConvertGRUSequenceMatcher;
+class TRANSFORMATIONS_API ConvertRNNSequenceMatcher;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief Converts opset4::(GRU/RNN/LSTM)Sequence to legacy (GRU/RNN/LSTM)SequenceIE.
```

Comment:
Please add such comment for every transformation class.

Reply:
done